packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,84 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  --terra-consumer-color: #64696c;
+  --terra-consumer-color-dark-30: #383f42;
+  --terra-consumer-color-dark-50: #1c1f21;
+  --terra-consumer-color-light-30: rgba(#383f42, 0.3);
+  --terra-consumer-color-light-50: rgba(#64696c, 0.5);
+  --terra-consumer-color-light-65: rgba(#383f42, 0.65);
```

Comment:
since`-terra-consumer-color-light-50` is used in both primary and secondary disabled buttons, so i didn't distinguish this set of intermediate variables with primary/secondary